components/cards/Cards.module.css

```diff
@@ -0,0 +1,52 @@
+.recipecard {
+  border: 3px solid var(--MainClr);
+  border-radius: 15px;
+  padding: 5px;
+  width: 80vw;
+  background-color: rgba(255, 255, 255, 0.75);
+  -webkit-box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.8);
+  -moz-box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.8);
+  box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.8);
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  gap: 0 0;
+  grid-template-areas:
+    "Img Img Name Name Name"
+    "Img Img Name Name Name"
+    "Img Img Name Name Name"
+    "Img Img Button Button Button";
+}
+
+.recipecardimgdiv {
```

Comment:
You should use camelCase, also this naming is very complex.
Again the Card in the class name is not necessary.

Reply:
would be fixed in next pullrequest. thank you